tests/lean/hashmap_on_disk/lakefile.lean

```diff
@@ -0,0 +1,22 @@
+import Lake
+open Lake DSL
+
+require mathlib from git
+  "https://github.com/leanprover-community/mathlib4.git"
+
+package «hashmap» {
+  -- add package configuration options here
+}
+
+lean_lib «HashmapMain» {
+  -- add library configuration options here
+}
+
+lean_lib «Base» {
+  -- add library configuration options here
+}
+
+@[default_target]
+lean_exe «hashmap» {
```

Comment:
Do we really need a lean_exe?

Reply:
right now we need it for the sake of running the tests, since I have yet to find a way to implement #assert in a way that it runs #eval then checks for `True` -- I find it convenient to have a placeholder, for a larger example, this would be a good place to add e.g. more unit-tests, performance benchmarks, etc.

Reply:
I guess that for the time being, whenever there are unit tests, in Lean, we could generate a `Main` file which groups all the unit tests, and declare it as `lean_exe` ?